### DISCUSSION TITLE
Is NS planned to support a web runtime? If so where can I help?

### DISCUSSION THREAD

**Discussion by wfzyx** [NativeScript/NativeScript#10622] [Ideas]
Even when web is not the main target of an app, having it running on web enables the developer to use browser dev tools to speed up development;

I looked around on the org and in the docs, and didn't really got a strong answer on this instance; 

Does anybody knows if this is something in the horizon, if not so; What would be a decent foundation to start coding such runtime integration?

I believe this was supposed to be the place with an answer, but its 404 https://docs.nativescript.org/code-sharing/index.html

**Comment by NathanWalker** [comment]
That link can be found here: https://docs.nativescript.org/guide/code-sharing 
Let us know where you got that other link and we can correct it.

## Possible right now

In general, allowing NativeScript to grow naturally alongside web specific markup in a workspace is pretty powerful today because there are literally 0 compromises. In a workspace you can drive both web and NativeScript apps with identical shared logic and component mechanics while decorating each view specifically tailored for the delivery platform. 
For example: https://nstudio.io/blog/nx-javascript-is-dead

## What about singular view markup?

NativeScript markup can be rendered as web (what we generally refer to as the inverse/reverse approach - similar to RN for web) which experiment was done a couple years ago:
https://github.com/NathanWalker/nativescript-angular-web-components

In general, works fine. It's just that you typically want very acute control over web markup specifically where a workspace driven approach provides ultimate control.

## Provide feedback

In general, you might briefly outline what you feel would be valuable for your dev building purposes (an ideal) and discuss from there; collaboration with you on such a thing would be awesome, thanks for starting the discussion!
